src/views/Viewer.vue

```diff
@@ -598,25 +617,19 @@ export default {
 
 		showSidebar() {
 			// Open the sidebar sharing tab
-			OCA.Files.App.fileList.showDetailsView(this.currentFile.basename)
+			OCA.Files.Sidebar.open(this.currentFile.filename)
 			this.showAppsSidebar()
 		},
 
 		showAppsSidebar() {
 			this.isSidebarShown = true
-			const sidebar = document.getElementById('app-sidebar')
-			if (sidebar) {
-				sidebar.classList.add('app-sidebar--full')
-			}
-
-			// overriding closing function
-			const origHideAppsSidebar = OC.Apps.hideAppSidebar
-			OC.Apps.hideAppSidebar = ($el) => {
-				this.hideAppsSidebar()
-				origHideAppsSidebar($el)
-			}
-
-			this.sidebarWidth = sidebar.offsetWidth
+			setTimeout(() => {
```

Comment:
we really need to bind the sidebar open to a proper Promise so that apps can await for it